assets/src/styles/blocks/Columns.scss

```diff
@@ -16,6 +16,7 @@
     font-size: $font-size-lg;
     font-family: $roboto;
     font-weight: 700;
+    color: var(--columns-header--color, $grey-80);
```

Comment:
I also switch these up all of the time, should be `heading` instead :smile:

The format is good, but maybe we should think about how to handle variables that apply to a block. If we consistently use a prefix for each block, then we could automatically extract all variables for a given block. Do you think it would make sense to separate this prefix too with 2 dashes? So that the resulting variable would be 
```css
color: var(--block-columns--column-heading--color, $grey-80);
```

Reply:
The result is a bit long, but that wouldn't be an issue when we start using shorthand notation.
```css
--block-columns--column-heading-- {
  color: $grey-80;
}
```

Reply:
Updated!